runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangePlugin.java

```diff
@@ -372,6 +406,22 @@ public void onMenuEntryAdded(MenuEntryAdded event)
 		// At the moment, if the user disables quick lookup, the input listener gets disabled. Thus, isHotKeyPressed()
 		// should always return false when quick lookup is disabled.
 		// Replace the default option with "Search ..." when holding alt
+		final String option = Text.removeTags(event.getOption()).toLowerCase();
+		final String target = Text.removeTags(event.getTarget()).toLowerCase();
+
+		//if shift key is pressed, primary menu option is changed from 'view offer' to 'abort offer'
+		if (config.enableShiftAbort())
+		{
+			if (shiftPressed)
+			{
+				if (option.equals("view offer"))
+				{
+					swap("abort offer", option, target, true);
```

Comment:
There is probably an easier way to do this than copying all of the swap/search logic from the menu entry swapper.

Reply:
Easiest is probably to just put it to menu entry swapper :p

Reply:
Not sure if it really fits in menu entry swapper, but I'm happy to move it there @deathbeam

Reply:
Well, it swaps menu entries. And also, you had to copy bunch of code from menu entry swapper to this plugin, so that kind of already tells that this is wrong place for it

Reply:
Good point @deathbeam I'll get it moved over :D